src/hooks/useTranslations/index.ts

```diff
@@ -0,0 +1,14 @@
+import { stardust } from '@nebula.js/stardust';
+import registerLocale from '../../locale/src/index';
+
+interface UseTranslationsArgs {
+  translator: stardust.Translator;
+}
+
+const useTranslations = ({ translator }: UseTranslationsArgs): stardust.Translator => {
```

Comment:
This is not a hook. As it's not bound by the rules of hooks.

I think you should make it into a hook tho. As currently `registerLocale` is called everytime the HeadCellMenu component is rendered.

Reply:
Yes, that is correct. I'll try to wrap it around `useMemo` from react